### ISSUE TITLE
get_columns_comments() not working

### ISSUE THREAD

**Issue by samliuq** [awslabs/aws-data-wrangler#767]
**Describe the bug**
```python
wr.catalog.get_columns_comments(database='ocean_qualiu'
                                , table='test_wr_2_test_parquet_write_a'
                                , boto3_session = boto3.session.Session(region_name = 'us-east-1'))
```

**Error Message: **
```bash
---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
<ipython-input-8-a4197a0b32f5> in <module>
----> 1 wr.catalog.get_columns_comments(database='ocean_qualiu', table='test_wr_2_test_parquet_write_a', boto3_session = boto3.session.Session(region_name = 'us-east-1'))

~/anaconda3/envs/python3/lib/python3.6/site-packages/awswrangler/_config.py in wrapper(*args_raw, **kwargs)
    415                 del args[name]
    416                 args = {**args, **keywords}
--> 417         return function(**args)
    418 
    419     wrapper.__doc__ = _inject_config_doc(doc=function.__doc__, available_configs=available_configs)

~/anaconda3/envs/python3/lib/python3.6/site-packages/awswrangler/catalog/_get.py in get_columns_comments(database, table, catalog_id, boto3_session)
    835     comments: Dict[str, str] = {}
    836     for c in response["Table"]["StorageDescriptor"]["Columns"]:
--> 837         comments[c["Name"]] = c["Comment"]
    838     if "PartitionKeys" in response["Table"]:
    839         for p in response["Table"]["PartitionKeys"]:

KeyError: 'Comment'

```

**Environment**

Provide your `pip list` output, particularly the version of the AWS Data Wrangler library you used. Providing this information may significantly improve resolution times.

**To Reproduce**

Steps to reproduce the behavior.

*P.S. Please do not attach files as it's considered a security risk. Add code snippets directly in the message body as much as possible.*

**Comment by jaidisido** [comment]
Good catch, it will be fixed in this PR #779

**Comment by robert-schmidtke** [comment]
@jaidisido @LeonLuttenberger this appears to be reintroduced in https://github.com/aws/aws-sdk-pandas/pull/1979
